testhelper/testhelper.go

```diff
@@ -80,6 +91,25 @@ func getStubLidar(t *testing.T) *inject.Camera {
 	return cam
 }
 
+func getStubIMU(t *testing.T) *inject.MovementSensor {
+	imu := &inject.MovementSensor{}
+	imu.LinearAccelerationFunc = func(ctx context.Context, extra map[string]interface{}) (r3.Vector, error) {
+		t.Error("TEST FAILED stub IMU LinearAcceleration called")
+		return r3.Vector{}, errors.New("invalid sensor")
+	}
+	imu.AngularVelocityFunc = func(ctx context.Context, extra map[string]interface{}) (spatialmath.AngularVelocity, error) {
+		t.Error("TEST FAILED stub IMU AngularVelocity called")
+		return spatialmath.AngularVelocity{}, errors.New("invalid sensor")
+	}
+	imu.PropertiesFunc = func(ctx context.Context, extra map[string]interface{}) (*movementsensor.Properties, error) {
+		return &movementsensor.Properties{
+			AngularVelocitySupported:    true,
```

Comment:
[q] should these be true for stub IMU?

Reply:
Yes, as otherwise creating the new IMU fails. Is there an alternative way of implementing this?